staking/economics/reward.go

```diff
@@ -12,6 +12,10 @@ import (
 	"github.com/harmony-one/harmony/numeric"
 )
 
+const (
+	numBlocksPerYear = 300_000_000
```

Comment:
We shouldn't hard code the block numbers per year. it's a dynamic number. Depending how you calculate the average block time (e.g. averaging over last 1000 blocks or 10000 blocks). Let's avg over last epoch's block time, instead of hard coded like this.

Reply:
ah, yes, my bad. I left it in while developing, will do avg